HerPortal.Data/DataAccessProvider.cs

```diff
@@ -45,6 +46,7 @@ public async Task<IEnumerable<User>> GetAllActiveUsersAsync()
         return await context.Users
             .Where(u => u.HasLoggedIn)
             .Include(u => u.LocalAuthorities)
+            .Include(u => u.Consortia)
```

Comment:
Same here. This one might not even need the original `Include(u => u.LocalAuthorities)`, but do check.

Reply:
I think this one can actually do without both. It seems to just need the email address and then uses the previous comment's Consortia and Local Authorities Includes when it needs those pieces of info.